fix: PPOM - What's New copy text refers to OpenSea previous feature - non existing in Mobile

blackdevelopa opened the pull request:
## **Description**

<!--
Write a short description of the changes included in this pull request, also include relevant motivation and context. Have in mind the following questions:
1. What is the reason for the change?
2. What is the improvement/solution?
-->

This PR updates the copy we show on the What's New modal for Blockaid. The new copy should mirror the image below. See [comment](https://github.com/MetaMask/metamask-mobile/issues/7782#issuecomment-1808550812) for ref.

<img width="506" alt="Screenshot 2023-11-13 at 13 49 17" src="https://github.com/MetaMask/metamask-mobile/assets/29962968/11eab3c7-9449-4043-9b16-51ce94a8758d">



## **Related issues**

Fixes: #7782 

## **Manual testing steps**

1. Include blockaid in .env
2. Start app

## **Screenshots/Recordings**

<!-- If applicable, add screenshots and/or recordings to visualize the before and after of your change. -->

### **Before**

![Screenshot from 2023-11-13 17-40-26](https://github.com/MetaMask/metamask-mobile/assets/54408225/fb9f1014-7af8-41f3-a667-9edc661a4aba)

### **After**

<img width="440" alt="Screenshot 2023-11-15 at 09 50 20" src="https://github.com/MetaMask/metamask-mobile/assets/29962968/fbd1649c-b4bd-4575-9c1b-ed97693358db">


## **Pre-merge author checklist**

- [x] I’ve followed [MetaMask Coding Standards](https://github.com/MetaMask/metamask-mobile/blob/main/.github/guidelines/CODING_GUIDELINES.md).
- [x] I've clearly explained what problem this PR is solving and how it is solved.
- [x] I've linked related issues
- [x] I've included manual testing steps
- [x] I've included screenshots/recordings if applicable
- [x] I’ve included tests if applicable
- [x] I’ve documented my code using [JSDoc](https://jsdoc.app/) format if applicable
- [x] I’ve applied the right labels on the PR (see [labeling guidelines](https://github.com/MetaMask/metamask-mobile/blob/main/.github/guidelines/LABELING_GUIDELINES.md)). Not required for external contributors.
- [x] I’ve properly set the pull request status:
  - [x] In case it's not yet "ready for review", I've set it to "draft".
  - [x] In case it's "ready for review", I've changed it from "draft" to "non-draft".

## **Pre-merge reviewer checklist**

- [x] I've manually tested the PR (e.g. pull and build branch, run the app, test code being changed).
- [x] I confirm that this PR addresses all acceptance criteria described in the ticket it closes and includes the necessary testing evidence such as recordings and or screenshots.

blackdevelopa committed:
update whats new copy for blockaid

jpuri commented:
@blackdevelopa : this is how it looks now:

![Screenshot 2023-11-14 at 11 19 10 PM](https://github.com/MetaMask/metamask-mobile/assets/2182307/6a4869e2-3bdb-4413-ad6d-0389e57ef947)

blackdevelopa committed:
move the title above the image

bschorchit commented:
Looks good to me!

blackdevelopa committed:
update

blackdevelopa committed:
Merge branch 'main' into 7782/update-whatsnew-copy-blockaid

blackdevelopa committed:
revert translations in 7782/update-whatsnew-copy-blockaid (#7824)

## **Description**

<!--
Write a short description of the changes included in this pull request,
also include relevant motivation and context. Have in mind the following
questions:
1. What is the reason for the change?
2. What is the improvement/solution?
-->

## **Related issues**

Fixes: #

## **Manual testing steps**

1. Go to this page...
2.
3.

## **Screenshots/Recordings**

<!-- If applicable, add screenshots and/or recordings to visualize the
before and after of your change. -->

### **Before**

<!-- [screenshots/recordings] -->

### **After**

<!-- [screenshots/recordings] -->

## **Pre-merge author checklist**

- [ ] I’ve followed [MetaMask Coding
Standards](https://github.com/MetaMask/metamask-mobile/blob/main/.github/guidelines/CODING_GUIDELINES.md).
- [ ] I've clearly explained what problem this PR is solving and how it
is solved.
- [ ] I've linked related issues
- [ ] I've included manual testing steps
- [ ] I've included screenshots/recordings if applicable
- [ ] I’ve included tests if applicable
- [ ] I’ve documented my code using [JSDoc](https://jsdoc.app/) format
if applicable
- [ ] I’ve applied the right labels on the PR (see [labeling
guidelines](https://github.com/MetaMask/metamask-mobile/blob/main/.github/guidelines/LABELING_GUIDELINES.md)).
Not required for external contributors.
- [ ] I’ve properly set the pull request status:
  - [ ] In case it's not yet "ready for review", I've set it to "draft".
- [ ] In case it's "ready for review", I've changed it from "draft" to
"non-draft".

## **Pre-merge reviewer checklist**

- [ ] I've manually tested the PR (e.g. pull and build branch, run the
app, test code being changed).
- [ ] I confirm that this PR addresses all acceptance criteria described
in the ticket it closes and includes the necessary testing evidence such
as recordings and or screenshots.
